fuzzers/lafintel/fuzzer.py

```diff
@@ -29,7 +29,9 @@ def prepare_build_environment():
     # (see https://github.com/php/php-src/blob/master/Zend/zend_cpuinfo.h).
     # It is not supported by clang-3.8, so we define the MACRO below
     # to replace any __builtin_cpu_supports() with 0, i.e., not supported
-    cflags = ['-O3', '-fPIC', '-D__builtin_cpu_supports\\(x\\)=0']
+    cflags = ['-O3', '-fPIC']
+    if 'php' in os.environ['PWD']:
```

Comment:
just wondering how robust relying on PWD is. perhaps we should create a BENCHMARK variable later, if benchmark specific options become more common.

Reply:
Yeah we will replace this with something more robust.